airmozilla/starred/tests/test_starred.py

```diff
@@ -0,0 +1,198 @@
+import json
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from funfactory.urlresolvers import reverse
+from nose.tools import eq_, ok_
+
+from airmozilla.main.models import Event
+from airmozilla.starred.models import (
+    StarredEvents,
+)
+
+
+class TestStarredEvent(TestCase):
```

Comment:
Change the base class from `django.test.TestCase` to `airmozilla.base.tests.testbase.DjangoTestCase`.